src/constants.ts

```diff
@@ -97,13 +101,15 @@ const blockchainNetworkMap = {
   [`${Blockchain.Polygon}:${NetworkId.Main}`]: 0b0001_0000 | 0b0000_0001,
   [`${Blockchain.Polygon}:${NetworkId.Mumbai}`]: 0b0001_0000 | 0b0000_0010,
   [`${Blockchain.Polygon}:${NetworkId.Amoy}`]: 0b0001_0000 | 0b0000_0011,
+  [`${Blockchain.Polygon}:${NetworkId.Zkevm}`]: 0b0011_0000 | 0b0000_0001,
```

Comment:
we shouldn't change Blockchain bytes, for Polygon it's `0b0001_0000`, you need to change NetworkId flag

Reply:
Or we shouldn't change `networkFlag` to not break anything ?
CC @vmidyllic

Reply:
we could set it like this `0b0001_0000 | 0b0010_0001,`

so first 4 bits would be the same as other Polygon and in netwrok we will keep final version of networkFlag as it was

Reply:
```golang
	{Blockchain: Polygon, NetworkID: Zkevm}:   0b0001_0000 | 0b0000_0100,
	{Blockchain: Polygon, NetworkID: Cardona}: 0b0001_0000 | 0b0000_0101,
```

Reply:
Fixed